src/test/java/com/epam/tc/hw5/common/UserTableContent.java

```diff
@@ -0,0 +1,161 @@
+package com.epam.tc.hw5.common;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class UserTableContent {
+    final WebDriver driver;
+    final WebDriverWait wait;
+    final List<WebElement> numberTypes;
+    final List<WebElement> usernames;
+    final List<WebElement> descriptions;
+    final List<WebElement> vipCheckboxes;
+    final List<WebElement> tableRows;
+    final List<WebElement> columnNames;
+    final WebElement dropdownElement;
+    final WebElement vipCheckbox;
+    final WebElement logPanel;
+
+    public UserTableContent(WebDriver driver) {
+        this.driver = driver;
+        this.wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        this.numberTypes = driver.findElements(By.xpath("//table[@id='user-table']//select"));
+        this.usernames = driver.findElements(By.xpath("//table[@id='user-table']//td/a"));
+        this.descriptions = driver.findElements(By.xpath("//table[@id='user-table']//td[4]//span"));
+        this.vipCheckboxes = driver.findElements(By.xpath("//input[@type='checkbox']"));
+        this.tableRows = driver.findElements(By.xpath("//table[@id='user-table']//tr"));
+        this.columnNames = driver.findElements(By.xpath("//table[@id='user-table']//th"));
+        this.dropdownElement = driver.findElement(By.xpath("//table[@id='user-table']//tr[1]//select"));
+        this.vipCheckbox = driver.findElement(By.cssSelector("#ivan"));
+        this.logPanel = wait.until(
+                ExpectedConditions.visibilityOfElementLocated(
+                        By.xpath("//div[contains(@class, 'info-panel-section')]")));
+    }
+
+    public int countOfNumberType() {
+        return numberTypes.size();
+    }
+
+    public boolean areNumberTypesDisplayed() {
+        boolean areNumberTypesDisplayed = true;
+        for (WebElement numberType : numberTypes) {
+            if (!numberType.isDisplayed()) {
+                areNumberTypesDisplayed = false;
+            }
+        }
+        return areNumberTypesDisplayed;
+    }
+
+
+    public int countOfUsernames() {
+        return usernames.size();
+    }
+
+    public boolean areUsernamesDisplayed() {
+        boolean areUsernamesDisplayed = true;
+        for (WebElement username : usernames) {
+            if (!username.isDisplayed()) {
+                areUsernamesDisplayed = false;
+            }
+        }
+        return areUsernamesDisplayed;
```

Comment:
Please use streams here and try to find any case that doesn't match the criteria, and return true/false based on this